src/main/java/de/retest/recheck/execution/RecheckAdapters.java

```diff
@@ -9,16 +9,23 @@
 
 	private static final ServiceLoader<RecheckAdapter> adapters = ServiceLoader.load( RecheckAdapter.class );
 
-	private RecheckAdapters() {
-
-	}
+	private RecheckAdapters() {}
 
 	public static RecheckAdapter findAdapterFor( final Object toVerify ) {
 		return StreamSupport.stream( adapters.spliterator(), false ) //
 				.filter( adapter -> adapter.canCheck( toVerify ) ) //
 				.findAny() //
-				.orElseThrow( () -> new UnsupportedOperationException( String.format(
-						"No recheck adapter registered that can handle an object of %s.", toVerify.getClass() ) ) );
+				.orElseThrow( () -> createHelpfulExceptionForMissingAdapter( toVerify.getClass().getName() ) );
+	}
+
+	protected static UnsupportedOperationException createHelpfulExceptionForMissingAdapter( final String className ) {
+		final String msg =
+				String.format( "No recheck adapter registered that can handle an object of class %s.", className );
+		if ( className.startsWith( "org.openqa.selenium" ) ) {
+			return new UnsupportedOperationException(
+					msg + "\n You need to add recheck-web (https://github.com/retest/recheck-web) to the classpath." );
```

Comment:
Maybe "Please use recheck-web (…) instead of recheck directly."? But we can also keep it as it is.

Reply:
Since it is possible, that this problem arises also when combining multiple rechecks (e.g. XML, web), I would keep it as is.